src/containers/Admin/ThemeTab/index.tsx

```diff
@@ -45,42 +87,112 @@ const ThemeTab = (): JSX.Element => {
+                på siden hvor du endrer på instillingene. Fargene vil
+                reflekteres både på tavla og i instillingene.
+            </Paragraph>
+            <Heading3 className="heading">Velg farger</Heading3>
+            <div className="theme-tab__grid">
                 <RadioCard
                     title="Entur (standard)"
                     cardValue="default"
                     preview={Entur}
-                    selected={radioValue === 'default'}
+                    selected={themeRadioValue === 'default'}
                     callback={(val): void => switchTheme(val as Theme)}
                     className="theme-tab__theme-card"
                 />
                 <RadioCard
                     title="Mørkt"
                     cardValue="dark"
                     preview={Dark}
-                    selected={radioValue === 'dark'}
+                    selected={themeRadioValue === 'dark'}
                     callback={(val): void => switchTheme(val as Theme)}
                     className="theme-tab__theme-card"
                 />
                 <RadioCard
                     title="Lyst"
                     cardValue="light"
                     preview={Light}
-                    selected={radioValue === 'light'}
+                    selected={themeRadioValue === 'light'}
                     callback={(val): void => switchTheme(val as Theme)}
                     className="theme-tab__theme-card"
                 />
                 <RadioCard
                     title="Grått"
                     cardValue="grey"
                     preview={Grey}
-                    selected={radioValue === 'grey'}
+                    selected={themeRadioValue === 'grey'}
                     callback={(val): void => switchTheme(val as Theme)}
                     className="theme-tab__theme-card"
                 />
             </div>
+            <div>
+                <Heading3 className="heading">Velg rotasjon</Heading3>
+                <Paragraph className="theme-tab__paragraph">
+                    Her kan du velge hvilken retning innholdet på tavla skal
+                    vises.
+                </Paragraph>
+                <div className="theme-tab__grid">
+                    <RadioCard
+                        title="Standard"
+                        cardValue="standard"
+                        preview={directionPreviewImages.Standard}
+                        selected={directionRadioValue === 'standard'}
+                        callback={(val): void =>
+                            switchDirection(val as Direction)
+                        }
+                        className="theme-tab__theme-card"
+                    />
+                    <RadioCard
+                        title="Rotert"
+                        cardValue="rotert"
+                        preview={directionPreviewImages.Rotated}
+                        selected={directionRadioValue === 'rotert'}
```

Comment:
Kanskje bruke en konstant her og over? I stedet for tekst standard og rotert

Reply:
Enig, forandret til å bruke enum i cardValue og selected